src/css.js

```diff
@@ -336,16 +333,23 @@ jQuery.each( [ "height", "width" ], function( i, name ) {
 		},
 
 		set: function( elem, value, extra ) {
-			var styles = extra && getStyles( elem );
-			return setPositiveNumber( elem, value, extra ?
-				augmentWidthOrHeight(
+			var matches = rcssNum.exec( value ),
```

Comment:
I'd rather add the verification. As far as the variable name, I think the use of `matches` for regex checks is older than `valueParts`, and is actually what was used in `setPositiveNumber`.